metricbeat/mb/metricset/http.go

```diff
@@ -0,0 +1,71 @@
+package metricset
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/elastic/beats/metricbeat/mb"
+)
+
+type Http struct {
+	mb.BaseMetricSet
+	client *http.Client // HTTP client that is reused across requests.
+}
+
+// NewHttpMetricset creates new instance of HttpMetriSset
```

Comment:
Looks more like the comment is out of date, it should be "creates a new instance of Http". Btw, HTTP is an acronym so usually it shows up as HTTP in Go code, right? That might look weird though.